packages/@expo/server/src/index.ts

```diff
@@ -1,21 +1,21 @@
 import './install';
 
-import type { ExpoRoutesManifestV1, RouteInfo } from 'expo-router/build/routes-manifest';
 import fs from 'node:fs';
 import path from 'node:path';
 
-import { ExpoRouterServerManifestV1FunctionRoute } from './types';
+import { ExpoError } from './error';
+import { InternalResponse, Manifest, RawManifest, Route } from './types';
 
 const debug =
   process.env.NODE_ENV === 'development'
     ? (require('debug')('expo:server') as typeof console.log)
     : () => {};
 
-function getProcessedManifest(path: string): ExpoRoutesManifestV1<RegExp> {
+function getProcessedManifest(path: string): Manifest {
   // TODO: JSON Schema for validation
-  const routesManifest = JSON.parse(fs.readFileSync(path, 'utf-8')) as ExpoRoutesManifestV1;
+  const routesManifest = JSON.parse(fs.readFileSync(path, 'utf-8')) as RawManifest;
```

Comment:
We talked in DMs about this, but just repeating here for documentation: We'd like this to be dynamic and shared, hence, this will have to accept the `routesManifest`, which could be passed directly to the creation function for a route handler.